server/start.go

```diff
@@ -241,52 +233,63 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 		return fmt.Errorf("error creating listener: %v", err)
 	}
 
-	svr.SetLogger(ctx.Logger.With("module", "abci-server"))
+	svr.SetLogger(svrCtx.Logger.With("module", "abci-server"))
 
-	err = svr.Start()
-	if err != nil {
-		fmt.Println(err.Error())
-		os.Exit(1)
-	}
+	ctx, cancelFn := context.WithCancel(context.Background())
```

Comment:
Q: Does `svrCtx` contains a Go's std context to pass here instead of new background?

Reply:
Nope :/